src/AlertsManagement/AlertsManagement/help/Az.AlertsManagement.md

```diff
@@ -0,0 +1,45 @@
+---
+Module Name: Az.AlertsManagement
+Module Guid: ab73cb67-5713-4179-b6d7-f76ab3ad148f
+Download Help Link: https://docs.microsoft.com/en-us/powershell/module/az.alertsmanagement
+Help Version: 1.0.0.0
+Locale: en-US
+---
+
+# Az.AlertsManagement Module
+## Description
+Manages alerts, smart groups and action rule configuration for Alerts Management Platform.
+
+## Az.AlertsManagement Cmdlets
+### [Measure-AzAlertStatistic](Measure-AzAlertStatistic.md)
+Gets the summary of alerts grouped by some properties.
+
+### [Get-AzAlert](Get-AzAlert.md)
+Gets list of alerts filtered on various parameters.
+
+### [Get-AzAlertHistory](Get-AzAlertHistory.md)
```

Comment:
The current analysis issue is the mismatch between this and the actual cmdlet name